posts/Martin/2015-07-13-introduction-to-hibernate-search-genericjpa.adoc

```diff
@@ -0,0 +1,460 @@
+
+== The example project
+
+Imagine a Book store (ok, a really simple one).
+Information about their books are stored in their MySQL database and accessed via EclipseLink.
+A sample persistence.xml and Maven pom.xml for that would look like this:
+
+[source,xml]
+----
+<persistence xmlns="http://java.sun.com/xml/ns/persistence"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://java.sun.com/xml/ns/persistence http://java.sun.com/xml/ns/persistence/persistence_2_0.xsd"
+	version="2.0">
+ 
+<persistence-unit name="EclipseLink_MySQL"
+	transaction-type="RESOURCE_LOCAL">
+	<provider>org.eclipse.persistence.jpa.PersistenceProvider</provider>
+	<class>org.hibernate.search.genericjpa.test.entities.Book</class>
+	<properties>
+		<property name="javax.persistence.jdbc.driver" 
+			value="com.mysql.jdbc.Driver" />
+		<property name="javax.persistence.jdbc.url" 
+			value="jdbc:mysql://localhost:3306/testingdb" />
+		<property name="javax.persistence.jdbc.user" 
+			value="hibernate_user" />
+		<property name="javax.persistence.jdbc.password"
+			value="hibernate_password" />
+		<property name="eclipselink.ddl-generation" 
+			value="drop-and-create-tables" />
+		<property name="eclipselink.logging.level" 
+			value="INFO" />
+		<property name="eclipselink.ddl-generation.output-mode"
+			value="both" />
+	</properties>
+</persistence-unit>
+ 
+</persistence>
+----
+
+[source,xml]
+----
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.hibernate</groupId>
+    <artifactId>hibernate-search-jpa-example</artifactId>
+    <version>0.2.0.0</version>
+    <packaging>jar</packaging>
+    <name>hibernate-search-jpa-example</name>
+ 
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-deploy-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
+                </configuration>
+            </plugin>
+ 
+            <plugin>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
+                    <encoding>UTF-8</encoding>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+ 
+ 
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
+ 
+         <!-- this is needed for adding search later -->
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-search-jpa</artifactId>
+            <version>0.2.0.0</version>
```

Comment:
Why four digits? And is it available in Maven Central? Or should one add the JBoss repo to the POM?